src/KafkaFlow.Serializer.ApacheAvro/KafkaFlow.Serializer.ApacheAvro.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj" />
+      <ProjectReference Include="..\KafkaFlow.Serializer\KafkaFlow.Serializer.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Apache.Avro" Version="1.10.1" />
+      <PackageReference Include="Confluent.Kafka" Version="1.5.3" />
+      <PackageReference Include="Confluent.SchemaRegistry" Version="1.5.3" />
+      <PackageReference Include="Confluent.SchemaRegistry.Serdes.Avro" Version="1.5.3" />
```

Comment:
should use the same Confluent version of other projects

Reply:
Confluent package was updated to version 1.5.3